src/schema/rules/tabular_data/motion.yaml

```diff
@@ -13,6 +13,7 @@ motionChannels:
   columns:
     name__channels: required
     component: required
+    reference_frame__motion: recommended
```

Comment:
I am not sure if this makes sense, or if it should be moved to the `*_channels.json` completly

Reply:
no, this is needed here :+1: